edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Docker.E2E.Test/AgentTests.cs

```diff
@@ -145,7 +143,12 @@ public async Task AgentStartsUpModules(TestConfig testConfig)
                 var dockerCommandFactory = new DockerCommandFactory(client, loggingConfig, configSource.Object, new CombinedDockerConfigProvider(Enumerable.Empty<AuthConfig>()));
                 IRuntimeInfoProvider runtimeInfoProvider = await RuntimeInfoProvider.CreateAsync(client);
                 IEnvironmentProvider environmentProvider = await DockerEnvironmentProvider.CreateAsync(runtimeInfoProvider, restartStateStore, restartManager);
-                var commandFactory = new LoggingCommandFactory(dockerCommandFactory, loggerFactory);
+
+                var logFactoryMock = new Mock<ILoggerFactory>();
+                var logMock = new Mock<ILogger<LoggingCommandFactory>>();
+                logFactoryMock.Setup(l => l.CreateLogger(It.IsAny<string>()))
+                    .Returns(logMock.Object);
+                var commandFactory = new LoggingCommandFactory(dockerCommandFactory, logFactoryMock.Object);
```

Comment:
Is there anywhere else we mock this `ILoggerFactory`?

Reply:
Yes, in LoggingCommandFactoryTest.  It is all correct in that test class.